src/estimators/bloom_filters.py

```diff
@@ -489,14 +501,33 @@ def __call__(self, sketch_list):
 class SurrealDenoiser(DenoiserBase):
   """A closed form denoiser for a list of Any Distribution Bloom Filter."""
 
-  def __init__(self, probability=None, flip_one_probability=None,
+  def __init__(self, epsilon=None, probability=None, flip_one_probability=None,
```

Comment:
Is it possible to remove the probability arg? I realize it will take some refactoring, but leaving it around causes some ugliness below. 

If we must keep it creating a classmethod that takes an epsilon and creates a SurrealDenoiser would be cleaner. 

Complex relationships between args is not good.